pkg/node/manager/manager_test.go

```diff
@@ -895,3 +904,97 @@ func TestNodeManagerEmitStatus(t *testing.T) {
 	assert.NoError(m.Start(context.Background()))
 	<-done
 }
+
+type mockUpdater struct{}
+
+func (m *mockUpdater) UpdateIdentities(_, _ cache.IdentityCache, _ *sync.WaitGroup) {}
+
+type mockTriggerer struct{}
+
+func (m *mockTriggerer) UpdatePolicyMaps(ctx context.Context, wg *sync.WaitGroup) *sync.WaitGroup {
+	return wg
+}
+
+func (s *managerTestSuite) TestNodeWithSameInternalIP(c *check.C) {
```

Comment:
Could you also add `ipcacheExpect("upsert", "...")` for the local router IP as well in this test?

Reply:
I can't add these checks as I need to run a real `ipcache` in these test. Otherwise there will be `upsert` events without any IP collision detection, so the bug won't trigger.